resources/views/events/view.blade.php

```diff
@@ -433,4 +507,4 @@ class="btn btn-link dropdown-toggle"
   @include('includes.modals.event-add-volunteer')
   @include('includes.modals.event-request-review')
 
-  @endsection
+@endsection
```

Comment:
A lot of the changes displayed in the diff here are whitespace changes, makes it a bit hard to review actual changes.  Wonder if there's an easy way to avoid that - let's have a think at next retro.

Reply:
I think there may be an ignore button somewhere in github's review for whitespace.  I don't find diff format great for reviewing, as I like to see the context, so I tend to review more complex PRs in my IDE (PhpStorm) where I get a side-by-side diff.  So maybe the answer is a different review tool?

As for this one, I think it's probably my IDE tidying up - the indentation was off in the original.